src/makedocs.jl

```diff
@@ -166,6 +166,8 @@ For example, if you have `foo.md` and `bar.md`, `bar.md` would normally be evalu
 Evaluation order among the `expandfirst` pages is according to the order they appear in the
 argument.
 
+**`default_meta`** can be set to TODO. TODO default `@meta` block. TODO
```

Comment:
Just bikeshedding this here. I was thinking maybe it could be `page_meta` or even just `meta`.

So, currently, this will only affect the Markdown pages, and for docstrings you'd still have to do `setdocmeta!`, right? I am now wondering if this should also apply to docstrings (with `setdocmeta!` providing a per-module override still).

Do you have any sense if the global at-meta stuff you'd like to apply for pages would also be useful for docstrings? Are they usually basically the same in your experience?

Reply:
We use the same `DocTestSetup` in `@meta` blocks in every `.md` file and also set it via `setdocmeta!`. I believe they would be basically the same, and having `default_meta`/`page_meta`/`meta` also set a default for `doctests` inside docstrings would then unify 2 of 3 places where in my experience one needs to set these

So it'd help! But maybe not quite enough, at least if one also wants to run doctests outside of building the documentation. We do that (via an explicit `doctest(MyPackage)` call in our CI) in order to get code coverage.

For details see also my comment https://github.com/JuliaDocs/Documenter.jl/issues/2126#issuecomment-2854253638